.github/workflows/publish.yaml

```diff
@@ -96,11 +102,36 @@ jobs:
           PR_LABELS: "${{ join(github.event.pull_request.labels.*.name) }}"
         run: dart pub global run firehose --validate ${{ fromJSON('{"true":"--use-flutter","false":"--no-use-flutter"}')[inputs.use-flutter] }}
 
+      - name: Get comment id
+        if: ${{ (hashFiles('output/comment.md') != '') && inputs.write-comments }}
```

Comment:
`hashFiles('output/comment.md'` is Github actions shorthand for "if `output/comment.md` exists"